KS.Fiks.IO.Arkiv.Client/Schema/sok.xsd

```diff
@@ -204,12 +205,45 @@
         </xs:restriction>
     </xs:simpleType>
 
-    <xs:simpleType name="respons_filter">
+    <xs:simpleType name="respons_type">
         <xs:restriction base="xs:string">
             <xs:enumeration value="noekler"/>
             <xs:enumeration value="minimum"/>
             <xs:enumeration value="utvidet"/>
         </xs:restriction>
     </xs:simpleType>
+
+    <xs:simpleType name="inkluder">
+        <xs:restriction base="xs:string">
+            <xs:enumeration value="oppbevaringssted"/>
+            <xs:enumeration value="arkivskaper"/>
+            <xs:enumeration value="klassifikasjonssystem"/>
+            <xs:enumeration value="arkiv"/>
+            <xs:enumeration value="arkivdel"/>
+            <xs:enumeration value="mappe"/>
+            <xs:enumeration value="registrering"/>
+            <xs:enumeration value="klasse"/>
+            <xs:enumeration value="noekkelord"/>
+            <xs:enumeration value="kryssreferanse"/>
+            <xs:enumeration value="referanseArkivdel"/>
+            <xs:enumeration value="part"/>
+            <xs:enumeration value="merknad"/>
+            <xs:enumeration value="referanseArkivdel"/>
+            <xs:enumeration value="presedens"/>
+            <xs:enumeration value="postadresse"/>
+            <xs:enumeration value="telefonnummer"/>
+            <xs:enumeration value="moetedeltaker"/>
+            <xs:enumeration value="dokumentbeskrivelse"/>
+            <xs:enumeration value="forfatter"/>
+            <xs:enumeration value="korrespondansepart"/>
+            <xs:enumeration value="avskrivning"/>
+            <xs:enumeration value="dokumentflyt"/>
+            <xs:enumeration value="referanseTilMoeteregistrering"/>
```

Comment:
Hvilken xsd ligger disse to i? Tror uansett ikke de trengs som inkluder.

Reply:
Fant de samme sted som resten tror jeg? Tatt bort